packs/classfeatures/arcane-school.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "classfeature",
         "description": {
-            "value": "<p>Most wizards acquire their knowledge of spells from a formal educational institution, such as the Arcanamirium or the Magaambya. At 1st level, you choose your arcane school, which grants you magical abilities.</p>\n<p>You gain additional spells and spell slots from the curriculum taught at the school you attended. Arcane schools are described in detail on page 198. Some wizards follow the school of unified magical theory, which attempts to forge a new school by studying independently and drawing information from a multitude of texts and tutors. Though a wizard with this approach lacks the focus of formal training, they have greater flexibility.</p>"
+            "value": "<p>Most wizards acquire their knowledge of spells from a formal educational institution, such as the Arcanamirium or the Magaambya. At 1st level, you choose your arcane school, which grants you magical abilities.</p>\n<p>You gain additional spells and spell slots from the curriculum taught at the school you attended. Arcane schools are described in detail @UUID[Compendium.pf2e.journals.JournalEntry.kzxu2dI7tFxv6Ix6.JournalEntryPage.wDukeO3euLEGn6FA#arcane-schools]{page 198}. Some wizards follow the school of unified magical theory, which attempts to forge a new school by studying independently and drawing information from a multitude of texts and tutors. Though a wizard with this approach lacks the focus of formal training, they have greater flexibility.</p>"
```

Comment:
Since we have a journal link, we probably don't need the page number. Perhaps remove the "Arcane schools are described in detail" sentence and add the link to the previous sentence?